apps/nextjs-website/src/lib/api.ts

```diff
@@ -137,18 +137,22 @@ export async function getTutorialLists(productSlug?: string) {
   return manageUndefinedAndAddProduct(props);
 }
 
+//TODO: remove this fallback when both of this tasks https://pagopa.atlassian.net/browse/DEV-1557 and https://pagopa.atlassian.net/browse/DEV-1524 are resolved
 export async function getWebinars(): Promise<readonly Webinar[]> {
-  return webinars;
+  return await getWebinarsProps();
```

Comment:
What about removing this function now? It is used in two pages ([1](https://github.com/pagopa/developer-portal/blob/b8429908150ae4ac5c6677bd71524410c06f6b52/apps/nextjs-website/src/app/webinars/%5BwebinarSlug%5D/page.tsx#L13), [2](https://github.com/pagopa/developer-portal/blob/b8429908150ae4ac5c6677bd71524410c06f6b52/apps/nextjs-website/src/app/webinars/%5BwebinarSlug%5D/questions/page.tsx#L10))